test/unit/org/apache/cassandra/cql3/validation/operations/InsertUpdateIfConditionCollectionsTest.java

```diff
@@ -992,14 +992,14 @@ public void testNonFrozenEmptyCollection() throws Throwable
                    row(true));
         assertRows(execute("UPDATE %s SET l = null WHERE k = 0 IF l != ?", list("bar")),
                    row(true));
-        assertRows(execute("UPDATE %s SET l = null WHERE k = 0 IF l < ?", list("a")),
-                   row(true));
-        assertRows(execute("UPDATE %s SET l = null WHERE k = 0 IF l <= ?", list("a")),
-                   row(true));
         assertRows(execute("UPDATE %s SET l = null WHERE k = 0 IF l IN (?, ?)", null, list("bar")),
                    row(true));
 
         // Does not apply
+        assertRows(execute("UPDATE %s SET l = null WHERE k = 0 IF l < ?", list("a")),
```

Comment:
Same question as before, this is change of behavior...

Reply:
I do not disagree the logic is.... broken.
But I thought we agreed not to make any changes in LWT not to break people? And to leave it to Accord to get it right, no?